backend/test_memory_task_generation.py

```
@@ -0,0 +1,607 @@
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)